cross/numpy/Makefile

```diff
@@ -0,0 +1,16 @@
+PKG_NAME = numpy
+PKG_VERS = 1.17.5
```

Comment:
1.18.x and greater fail to compile on `aarch64` architecture (due to an internal compiler error...)
Since bazarr says `>=1.12` I took the latest version which builds on all platforms.